packages/faceted-search/src/components/FacetedToolbar/FacetedToolbar.component.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Action from '@talend/react-components/lib/Actions/Action';
+import { getTheme } from '@talend/react-components/lib/theme';
+
+import { FACETED_MODE } from '../../constants';
+import cssModule from './FacetedToolbar.scss';
+
+const theme = getTheme(cssModule);
+
+const SwitchFacetedMode = ({ facetedMode, id, onClickAdvanced, onClickClassic, t }) => (
```

Comment:
I don't remember what you say before Kickoff, but we are ok that, for now, switch mode from basic to advanced do not convert badge to TQL query?

Reply:
Yep adv will not be convert in basic